sbc_compassion/tools/import_letter_functions.py

```diff
@@ -302,13 +454,26 @@ def _find_languages(env, img, pattern_center, line_vals):
     :param pattern_center: Center position of detected pattern
     :param dict line_vals: Dictonnary containing the data for a line
     (and the template)
+    :param bool test: Save the image of all the subimage
```

Comment:
Again bad name or bad comment. Comment should be `tells if the import is done in test mode` or variable name should be `save_images`